### PR TITLE
mpl: Fixes static init order across translation units

### PR DESCRIPTION
**Pull request by QuantamHD**
Fixes ASAN violations at Google

### CODE REVIEW

File: src/mpl/src/ParquetFP/src/basepacking.h [hunk 1]
```diff
@@ -89,10 +90,10 @@ class Dimension
   std::vector<float> haloX, haloY;
   std::vector<float> channelX, channelY;
 
-  static const float Infty;             // = 1e100;
-  static const float Epsilon_Accuracy;  // = 1e10;
-  static const int Undefined;           // = -1;
-  static const int Orient_Num;          // = 8;
+  static constexpr float Infty = std::numeric_limits<float>::max();
+  static constexpr float Epsilon_Accuracy = 1e10f;  // = 1e10;
```
Comment (maliberty) [inline src/mpl/src/ParquetFP/src/basepacking.h:94]: the comment is now redundant

**Review by maliberty — CHANGES_REQUESTED** [review]

**Comment by oharboe** [comment]
> Fixes ASAN violations at Google

What is ASAN?

**Comment by maliberty** [comment]
> > Fixes ASAN violations at Google
> 
> What is ASAN?

[Address Sanitizer](https://clang.llvm.org/docs/AddressSanitizer.html)

**Comment by QuantamHD** [comment]
@maliberty What's up with the weird CI failures. Seems like the jobs just get cancelled

**Pull request merged**